web/src/global-styles/web-content.scss

```
@@ -0,0 +1,21 @@
+.web-content {
+    &__link {
```

Any reason we shouldn't assign this to all links?
```suggestion
    a {
```

---

I think instead of a selector `web-content` would just need to set the `--link-color` variable though